m2cgen/assemblers/fallback_expressions.py

```diff
@@ -0,0 +1,43 @@
+from m2cgen import ast
+from m2cgen.assemblers import utils
+
```

Comment:
I think worth leaving a comment of a pydoc explaining when these fallbacks apply. Something like:
```
This module provides an implementation for a variety of functions expressed in library's AST. 
These AST-based implementations are used as fallbacks in case when the target language lacks native support for respective functions provided in this module.
```

Reply:
My fault! Great docstring, will add it.